modules/script/script-impl/index.d.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="global.d.ts" />
+
+declare const app: {
+    name: string;
+    version: string;
+    config: Record<string, string>;
+};
+
+declare const log: {
+    debug: (...args: unknown[]) => void;
+    info: (...args: unknown[]) => void;
+    warning: (...args: unknown[]) => void;
+    error: (...args: unknown[]) => void;
+};
+
+declare const __: {
+    newBean: <T = unknown>(bean: string) => T;
+    toNativeObject: <T = unknown>(beanResult: T) => T;
+    toScriptValue: <T = Object>(value: T) => T;
```

Comment:
```suggestion
    toScriptValue: <T = object>(value: T) => T;
```

Reply:
Good catch. That was serious misstype that I missed.